helm/chart-operator/templates/priorityclass.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if .Values.e2e }}
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: giantswarm-critical
+value: 1000000000
+globalDefault: false
+description: "This priority class is used by giantswarm kubernetes components."
+{{- end }}
```

Comment:
Since helmclient will no longer create the priority class we need this template for the tests.